GUIDELINES.md

```diff
@@ -0,0 +1,282 @@
+# Guidelines
+
+### Block Element Modifier (BEM)
+
+TNA frontend toolkit uses the [Block Element Modifier (BEM)](http://getbem.com/) methodology when naming CSS classes. This is designed to help developers understand how the different classes relate to each other.
+
+### Grid
+
+TNA frontend toolkit uses [Bootstrap 4.3.1 grid system](https://getbootstrap.com/docs/4.3/layout/grid/). Other Bootstrap components are not included.
+
+### Namespacing and naming BEM classes
+
+* Prefix global components with with `tna-` to reduce the likelihood of conflicting with existing classes in applications..
+* Names are written in lowercase letters.
+* Words within the names of BEM entities are separated by a hyphen (-).
+* The element name is separated from the block name by a double underscore (__).
+* Modifiers are separated from the name of the block or element by a double hyphen (--).
+* The value of a modifier is separated from its name by a double hyphen (--).
+```html
+tna-[block]__[element]--[modifier]--[modifier-value]
```

Comment:
Very minor question: is `--[modifier]--[modifier-value]` a TNA convention? Are there situations where it's more flexible than `--[modifer]`? It just caught my eye because I haven't seen it before.

Some examples from other docs which just use `--[modifer]`:
* `block--size-big` in the [Get BEM](http://getbem.com/naming/) docs
* `app-button--inverse` in the [GOV.UK Design System](https://design-system.service.gov.uk/get-started/extending-and-modifying-components/#small-modifications-to-components).

Reply:
Sorry I didn't make it to the discussion, by the way! We're using BEM on Transfer Digital Records, so it should help us if you use it in the design system. 👍

Reply:
@suzannehamilton `modifier-value` is optional, but designed so a value could be used in specific scenarios. For example `tna-card__image--opacity--50` and `tna-card__image--opacity--75`. This modifier with value allows the card image to have an opacity of 50%. or 75%.

Reply:
@suzannehamilton It's from Two Dash naming convention https://en.bem.info/methodology/naming-convention/#two-dashes-style